easybuild/easyconfigs/c/Cube/Cube-4.3.4.eb

```diff
@@ -26,19 +26,12 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 sources = [SOURCELOWER_TAR_GZ]
 source_urls = ['http://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
-
-checksums = [
-    '50f73060f55311cb12c5b3cb354d59fa',     # cube-4.3.4.tar.gz
-]
-
-# builddependencies = [ ('Qt', '4.8.6'), ]
-dependencies = [ ('Qt', '4.8.6'), ]
-preconfigopts = 'export PATH=$EBROOTQT/bin:$PATH && '
-# configopts = ' QT_PATH=$EBROOTQT/bin' # '--build=x86_64-pc-linux-gnu' # ' --with-gui --with-qt'
+checksums = [ '50f73060f55311cb12c5b3cb354d59fa' ]
+modextrapaths = { 'LD_LIBRARY_PATH': '../../../../../../../../../../../../../../../../../../usr/lib64' } # prevents conflict with DDT's Qt
 
 sanity_check_paths = {
-    'files': ["bin/cube", ("lib/libcube4.a", "lib64/libcube4.a"),
-              ("lib/libcube4.%s" % SHLIB_EXT, "lib64/libcube4.%s" % SHLIB_EXT)],
+    'files': ["bin/cube", ("lib/libcube4.a", "lib64/libcube4.a"), 
+                          ("lib/libcube4.so", "lib64/libcube4.so")],
```

Comment:
why changing this? If it was working before, please keep the generic way of referring to shared libraries.